Add Parameter Documentation for Subducting Plate Feature

danieldouglas92 opened the pull request:
Update the manual by adding the section for the subducting plate feature in parameter documentation, addresses a point in Issue #766.

MFraters reviewed (commented):
Thanks for writing this up, very nice! Also cool to see how you made the figures to explain the concepts. I have a few small comments, but otherwise looks good to me.

1. I guess you are trying to get into math mode with with the $ around some variables, but it doesn't render correctly. Maybe we to enable an extra plugin in sphinx?
2. Currently there is a limitation that the number of segements need to be the same in all sections. Could you add that?
3. In the downdip_sections.png figure, the black cross-sections are a bit hard to see. Maybe a lighter color or an outline would help.

danieldouglas92 commented:
@MFraters Thanks for the quick review! I addressed points 1 and 2, I just made M and N italicized, I don't think we need to worry about the math mode extension for sphinx at least for this PR. For point 3 I'll remake the figures later tonight, I also think the font should be a little larger, so I will comment at you again when it's ready for you take another look.

MFraters commented:
@danieldouglas92 Is this ready for another review?

danieldouglas92 commented:
@MFraters Yes it is!

MFraters reviewed (approved):
Thanks for writing this up!
